docs/contributing/release_guide.rst

```diff
@@ -70,17 +48,57 @@ Pre-release
 * Open a pull request to update :file:`docs/about/citation.rst` to
   reflect the new version, and include the reserved DOI.
 
-.. Should we switch to Citation File Format?  It appears to be better
-   supported by Zenodo.
+  .. todo::
+
+     Should we switch to Citation File Format? It appears to be better
+     supported by Zenodo.
 
 * Open a pull request to update and alphabetize the author list in
-  :file:`docs/about/credits.rst`. Add ORCID_ identifiers, if needed.
+  :file:`docs/about/credits.rst`. Missing ORCID_ identifiers may be
+  added.
 
 * Open a pull request to update :file:`codemeta.json`. Update the author
   list, version, and other metadata, as needed. Update the
   ``"identifier"`` tag with the DOI for the new release.
 
-* Open a pull request to update :file:`.mailmap`. (Add bash command for this?)
+* Open a pull request to update :file:`.mailmap`.
+
+  .. todo::
+
+     Add a Python script here to update :file:`.mailmap`.
+
+.. Use ``git shortlog -nse | cut -f 2 | vim -c "sort" -c "vsplit .mailmap" -c
+   "windo diffthis"`` to compare the old and new :file:`.mailmap` version. Make sure
+   the old addresses are preserved in the new version, then overwrite the
+   existing :file:`.mailmap` file.
+   This part may not be all that relevant anymore, except if we're using ``git
+   shortlog``. ← put this in pre-release?
+
+Generate changelog
+==================
+
+* Create a pull request to revise changelog entries to make sure that
+  they are categorized correctly, understandable, and necessary.
+
+  .. tip::
+
+     Apply the :guilabel:`No changelog entry needed` label to pull
+     requests that change multiple changelog entries in order to skip
+     the changelog entry check.
+
+Perform code quality checks
+===========================
+
+* Open a pull request to ``main`` to update black_ and other pre-commit_
+  hooks to their most recent versions in :file:`.pre-commit-config.yaml`
+  for major and minor releases. Apply all changes by running:
+
+  .. code-block:: bash
+
+     pre-commit run --all-files
+
+* Open a pull request to re-execute pre-executed notebooks, such as
+  those for charged particle radiography.
```

Comment:
Sure feels like doing all of these via PRs would extend the duration of every release. If we wanna release more often, we need to be going in the opposite direction. A single PR should be good enough, no?

Reply:
My motivation for separate PRs is that the dominant timescale is often code reviews, and code review time often scales more than linearly with respect to PR size.  If someone else is available for rapid code reviews, I'd expect that multiple focused PRs would be faster so long as they're not interdependent.